feedback/views.py

```diff
@@ -505,11 +505,18 @@ def get_context_data(self, **kwargs):
         update_context_for_feedbacks(self.request, context)
         return context
 
+    def get_paginate_by(self, queryset):
```

Comment:
We should avoid copying the same method into multiple classes. There are two classes and perhaps there will be more some day. You can create a new mixin class (`class PaginatedMixin`) that contains the `get_paginate_by()` method. These two paginated classes can then inherit the mixin: 
`class ManageNotRespondedListView(ManageCourseMixin, PaginatedMixin, ListView)`